src/alert-manager/src/alert-handler/controllers/kubernetes.js

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+const k8s = require('@kubernetes/client-node');
+const kc = new k8s.KubeConfig();
+const logger = require('@alert-handler/common/logger');
+
+// clean TTL 24 hours jobs
+const cleanTTL24HJobs = () => {
+  logger.info('Cleaning completed TTL 24h jobs...');
+
+  const k8sApi = kc.makeApiClient(k8s.BatchV1Api);
+  k8sApi
+    .listNamespacedJob(
+      'default',
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      'time-to-live=24h', // labelSelector
```

Comment:
Can we change the label to `job-type=fix-nvidia-gpu-low-perf`? 'time-to-live=24h' is too general

Reply:
> Can we change the label to `job-type=fix-nvidia-gpu-low-perf`? 'time-to-live=24h' is too general

Used two labels `time-to-live=24h` & `created-by=alert-handler` to avoid being too general or too specific.